### DISCUSSION TITLE
Nice work

### DISCUSSION THREAD

**Discussion by bert07** [valinet/ExplorerPatcher#580] [General]
I couldnt get rid of the language bar en de settings  Icon though. But now at least I can work with Windows 11.

And if the people behind openshell make that program also ready for Windows 11, perhaps we kan tweak the file explorer as well!

Nice job. Thanks!

**Comment by valinet** [comment]
Thanks. Great to hear it’s working well.

**Reply by avimar** [reply]
Similar- the "always combine windows" was driving me nuts with several chome windows open it was a huge pain to switch between them.
Supposedly most of the registry hacks no longer work.

But your Explorer Patcher worked. Thanks!!!

**Reply by valinet** [reply]
Windows 11 has an altogether new taskbar. ExplorerPatcher reactivates the Windows 10 one, whose code is still mostly in explorer.exe.

**Comment by bert07** [comment]
The program also has an unexpected  advantage. In Win11 I couldn't get T-Clock Redux working any more. After installing Explorer Patcher it works again!

**Reply by valinet** [reply]
Yeah, it enables the good old Win32 Windows 10 taskbar, so **everything** that worked with that *should* work with it. Emphasis on *should*.

**Comment by yasin-uzun** [comment]
Awesome tool. Thanks developers!

**Comment by leik-software** [comment]
I love this tool! Makes Windows 11 great again 😁

**Comment by Anixx** [comment]
OpenShell works in Windows 11.
